SpatialGDK/Source/Public/Schema/Component.h

```diff
@@ -12,4 +12,33 @@ struct Component
 	bool bIsDynamic = false;
 };
 
+class ComponentStorageBase
```

Comment:
What do you think about namespacing this with `improbable`?

Reply:
it is in the improbable namespace already right?